src/socketio.quiver.json

```diff
@@ -6,6 +6,12 @@
       "socketio.quiver.js"
    	]
   },
+  "dependencies": {
+    "pgp-e2e": {
+      "url": "../freedom-pgp-e2e/pgpapi.json",
+      "api": "crypto"
+    }
+  },
   "default": "social2",
   "provides": [
     "social2"
```

Comment:
Can you also add the "pgpKeyName" to the args for login in this file?  Otherwise Freedom drops that arg when passed from uProxy to freedom-social-quiver, thanks